### ISSUE TITLE
Nextcloud 25 support

### ISSUE THREAD

**Issue by MerryMack22** [pdsinterop/solid-nextcloud#110]
I was attempting to install this app on my installation of Nextcloud (old install, not a snap) and received a message that it is not compatible with my version (Nextcloud 25.0.2, php 8.1).

Is support for this version in the pipeline or is there some way I can contribute to this goal?

Thanks

**Comment by Potherca** [comment]
Hi! Thank you for reporting this.

NC25 and PHP8.1 _should_ already work but the `solid/appinfo/info.xml` was not (yet) updated to support this. Hence the incompatibility error.

We are currently testing Solid-Nextcloud with NC25 to see if everything works as expected, once this is done, a new release will be rolled out with NC25 support.

**Comment by Potherca** [comment]
With the completion of #111 the automated test all pass. Well continue manual testing, if that passes as well, I'll roll out a new (minor) release and publish it to the NC store.

**Comment by Potherca** [comment]
Tested and resolved by #114

**Comment by Potherca** [comment]
@MerryMack22 Version `v0.7.2` (with NC25 support) has been released to the Nextcloud App store.

The install should now "just work", please open a new issue if you run into any problems.
